lib/booker/v5/internal_client.rb

```diff
@@ -0,0 +1,20 @@
+module Booker
+  module V5
+    class InternalClient < Booker::Client
+      include ::Booker::RequestHelper
+
+      V5_PREFIX = '/v5'
+      V5_INTERNAL_LOCATIONS_PREFIX = "#{V5_PREFIX}/internal/locations"
```

Comment:
Add freeze; the one on line 6 is probably frozen already since it's a string literal.

Reply:
@mstahl yes, sure. thank you for noticing it!